includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -393,13 +398,18 @@ private function maybe_add_lightbox_attributes( $attributes, $node ) {
 			$attributes['tabindex']          = 0;
 
 			$this->maybe_add_amp_image_lightbox_node();
+
+			if ( $is_node_wrapped_in_anchor ) {
+				// Remove the <a> that the image is wrapped in, as it can prevent the lightbox from working.
```

Comment:
Did you check if this issue has been reported in AMP? Perhaps it is a bug in AMP itself that should be fixed?

Reply:
Ah, that's a good point. It is a little strange that the plugin has to remove the `<a>`.

Reply:
I couldn't find an issue in github.com/ampproject/amphtml for this, but I'll keep looking.

Reply:
I also might open an issue in `amphtml` for this.

Reply:
I opened an issue in `amphtml`: https://github.com/ampproject/amphtml/issues/25021